### PR TITLE
Run golangci-lint on EE packages if present

### PR DESCRIPTION
**Pull request by agnivade**
#### Summary
The Go toolchain doesn't work with symlinks. Hence we need to explicitly
mention the directory name for it to work.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -161,6 +161,9 @@ golangci-lint: ## Run golangci-lint on codebase
 
 	@echo Running golangci-lint
 	golangci-lint run
```
Comment (hanzei) [inline Makefile]: Nit: Can we replace this with
```suggestion
	golangci-lint run ./...
```

This make it a bit more clear to me, what it does.

**Comment by hanzei** [comment]
The build failed because of issue with EE. Will fix them.

**Comment by agnivade** [comment]
```
golangci-lint run ./enterprise/...

../enterprise/ldap/ldap_session_test.go:397: File is not `gofmt`-ed with `-s` (gofmt)

		"p0": []string{"u0", "u1", "u2", "u3", "u4"},

		"p1": []string{"u5", "u6", "u7", "u8", "u9"},

		"p2": []string{"u10", "u11"},
```

Does this mean we need to fix _all_ pending golangci issues with the EE codebase before merging this ?

**Comment by hanzei** [comment]
@agnivade This is the last issue on the enterprise repo. I have opened a PR to fix this, which hopefully will be merged soon.

**Pull request merged**